rebar.config

```diff
@@ -31,13 +31,12 @@
 {cover_opts, [verbose]}.
 {cover_export_enabled, true}.
 
-{plugins, [coveralls]}.
-
 {erl_first_files, ["src/emqx_logger.erl"]}.
 
 {profiles,
  [{test,
-   [{deps,
+   [{plugins, [{coveralls, {git, "https://github.com/emqx/coveralls-erl", {branch, "github"}}}]},
```

Comment:
Good job!
is this the fix ? https://github.com/emqx/coveralls-erl/commit/07c21d0ca08eacb0727da82c9a6c07437307fccc
maybe consider sending it as a PR to upstream.

Reply:
Yes, Good idea :)